internal/tls/option.go

```diff
@@ -23,47 +23,6 @@ type Option func(*credentials) error
 
 var (
 	defaultOpts = []Option{
-		WithTLSConfig(&tls.Config{
```

Comment:
why do we move this block to tls.go?

Reply:
To avoid same pointer between `want` and `got` in test